### PR TITLE
[MM-15170] Fix admin console urls and system-scheme permission

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Fixes license check for `/permissions/system-scheme` url and url inconsistencies related to the group links in system console.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-15170

### CODE REVIEW

File: components/admin_console/group_settings/group_row.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default class GroupRow extends React.Component {
         }
         if (this.props.has_syncables) {
             return (
-                <Link to={'/admin_console/access-control/groups/' + this.props.mattermost_group_id}>
```
Comment (mgdelacroix) [inline components/admin_console/group_settings/group_row.jsx]: Unrelated to the ticket, but was causing the same problem for group links

**Comment by jwilander** [comment]
@mgdelacroix a conflict needs to be fixed

**Comment by bradjcoughlin** [comment]
This should have been fixed in my last PR #2656 , but looks like it became out-of-sync somehow.

**Comment by mgdelacroix** [comment]
> This should have been fixed in my last PR #2656 , but looks like it became out-of-sync somehow.

I saw it when trying to merge, we fixed it in the exact same way so I changed the commit name to include only the url fix and rebased on top of your change.

**Pull request merged**